src/input/input.stories.ts

```diff
@@ -26,6 +26,7 @@ const Template = (args) => ({
 		{{label}}
 		<input
 			cdsText
+			type="text"
```

Comment:
```suggestion
```

The type of the input is text by default, so we don't need to set the type here.

Reply:
yes, but the textInputContent projection selector requires`input[type=text]`, not setting the type will cause the input element to be placed inside labelContent instead of textInputContent.

Reply:
You're right, totally forgot about that. Could you add `cdsText` and `ibmText` to the the select options for `text-input-label`? This is to make it more flexible.

So it would be:
`<ng-content select="[cdsText],[ibmText],input[type=text],div"></ng-content>`

Reply:
done! I did the same with textarea.